lib/tasks/route_import.rake

```diff
@@ -5,12 +5,72 @@ require 'csv'
 namespace :routes do
   task :import, [:csv_file] => :environment do |_, args|
     Route.delete_all
+    BusStopsRoute.delete_all
+    @route_hash = {}
     CSV.foreach(args[:csv_file], headers: true, col_sep: ';') do |row|
       stop = BusStop.find_by hastus_id: row['stp_identifier']
       if stop.present?
-        route = Route.find_or_create_by number: row['rte_identifier'].strip,
-                                        description: row['rte_description']
+        route = Route.find_or_create_by number: row['rte_identifier'].strip
         stop.routes << route
+
+        direction = row['direction']
+        variant = row['variant']
+        stop_id = row['stp_identifier']
+        rank = row['stop_variant_rank']
+
+        @route_hash[route] ||= {}
+        @route_hash[route][direction] ||= {}
+        @route_hash[route][direction][variant] ||= []
```

Comment:
Could you combine these?

Reply:
Into what

Reply:
Do you understand what's happening on each line?